src/interactivity/base/EventSynthesis.cpp

```diff
@@ -166,64 +120,35 @@ std::deque<std::unique_ptr<KeyEvent>> Microsoft::Console::Interactivity::Synthes
 // alt + numpad
 // Note:
 // - will throw exception on error
-std::deque<std::unique_ptr<KeyEvent>> Microsoft::Console::Interactivity::SynthesizeNumpadEvents(const wchar_t wch, const unsigned int codepage)
+void Microsoft::Console::Interactivity::SynthesizeNumpadEvents(const wchar_t wch, const unsigned int codepage, InputEventQueue& keyEvents)
```

Comment:
An out-parameter allows us to directly append to an existing buffer instead of returning a buffer and then appending that buffer to another buffer. This function is only called from a single place too (and always has been: the clipboard handler).